### PR TITLE
Add suport for SPM

### PR DESCRIPTION
**Pull request by tomas-bat**
# :pencil: Description
- This PR updates the framework and adds support for Swift Package Manager.

# :bulb: What’s new?
- Added `Package.swift`

### CODE REVIEW

File: MluviiChat/MluviiChat/MluviiChatLibrary.swift [hunk 1]
```diff
@@ -73,7 +77,7 @@ public class MluviiChat :  UIViewController, WKUIDelegate, WKNavigationDelegate,
         return encodedLink!
     }
     
-    public func createWebView(url:String, companyGuid:String, tenantId:String, presetName:String?, language:String?) -> WKWebView {
+    public func createWebView(url:String, companyGuid:String, tenantId:String, presetName:String?, language:String?, scope:String?) -> WKWebView {
```
Comment (tommassbrand) [inline MluviiChat/MluviiChat/MluviiChatLibrary.swift:80]: Format :) add whitespace

File: MluviiChat/MluviiChat/MluviiChatLibrary.swift [hunk 2]
```diff
@@ -54,7 +54,7 @@ public class MluviiChat :  UIViewController, WKUIDelegate, WKNavigationDelegate,
         webView?.frame = CGRect(x:0, y: 0, width: size.width, height: size.height)
     }
     
-    public func createLink(url:String, companyGuid:String, tenantId:String, presetName:String? = nil, language:String? = nil) -> String {
+    public func createLink(url:String, companyGuid:String, tenantId:String, presetName:String? = nil, language:String? = nil, scope:String? = nil) -> String {
         var link: String = "https://\(url)/MobileSdkWidget?c=\(companyGuid)&t=\(tenantId)"
          var optionalPresetName = ""
          if(presetName != nil){
```
Comment (tommassbrand) [inline MluviiChat/MluviiChat/MluviiChatLibrary.swift:60]: Format the lines :)

File: MluviiChat/MluviiChat/MluviiChatLibrary.swift [hunk 3]
```diff
@@ -64,7 +64,11 @@ public class MluviiChat :  UIViewController, WKUIDelegate, WKNavigationDelegate,
         if(language != nil){
             optionalLanguage = "&l=\(language!)"
         }
-        let optionalQuery = "\(optionalPresetName)\(optionalLanguage)"
+        var optionalScope = ""
+        if(scope != nil){
+            optionalScope = "&s=\(scope!)"
+        }
```
Comment (tommassbrand) [inline MluviiChat/MluviiChat/MluviiChatLibrary.swift:70]: Format the lines and conditions :)

**Review by tommassbrand — CHANGES_REQUESTED** [review]

**Comment by tomas-bat** [comment]
@tommassbrand You commented on code from Mluvii's library, do we want to edit that?

**Comment by tommassbrand** [comment]
Oh okay, then let it be :) @tomas-bat

**Review by pchmelar — CHANGES_REQUESTED** [review]
- I don't know why this PR is showing changes in some files, that remain unchanged (like `MluviiChat.podspec` etc). 🤔 Probably something with how forks work on GitHub.
- `.DS_Store` and `xcuserdata` shouldn't be a part of the repo, please remove them and add them to `.gitignore` (you can copy-paste `.gitignore` from our DevStack repo) 
- Did you take a look at that SwiftUI version of this library? 🙂

**Comment by tomas-bat** [comment]
- Removed `.DS_Store` and `xcuserdata` directory. It is still showing one of the files in this directory as edited, not deleted (not sure why, guess it's because it was also changed), but that file is not in this branch anymore.
- I did not look into the SwiftUI version very much, I just wanted to update it, add the SPM support, so we can use it as is. Should I create another task as part of the refactor, that looks into whether it can be replaced with the SwiftUI version? @pchmelar

**Comment by pchmelar** [comment]
Yeah, please create a low-priority task for testing the SwiftUI version 🙂

**Review by pchmelar — APPROVED** [review]

**Comment by tomas-bat** [comment]
> Yeah, please create a low-priority task for testing the SwiftUI version 🙂

Created the task [here](https://mateedevs.atlassian.net/jira/software/projects/UP/boards/1/backlog?selectedIssue=UP-650).

**Pull request merged**